src/App.tsx

```diff
@@ -43,16 +43,20 @@ export const App: React.FC = () => {
     return true;
   });
 
+  const handleFilter = (actualFilter: FilterType) => {
+    setFilter(actualFilter);
+  };
+
   if (!USER_ID) {
     return <UserWarning />;
   }
 
   return (
     <div className="todoapp">
-      <TodoContent todos={todos} filter={filter} setFilter={setFilter}>
+      <TodoContent todos={filteredTodos}>
         <TodoList todos={filteredTodos} />
         {todos.length !== 0 && (
-          <Footer todos={todos} filter={filter} setFilter={setFilter} />
+          <Footer todos={todos} filter={filter} handleFilter={handleFilter} />
```

Comment:
[checklist](https://github.com/mate-academy/react_todo-app-loading-todos/blob/master/checklist.md#basic-react-checklist): Follow [these](https://medium.com/javascript-in-plain-english/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364) naming conventions for handleFilter